### DISCUSSION TITLE
Separate permission for /is setperms nickname

### DISCUSSION THREAD

**Discussion by exzolink** [BG-Software-LLC/SuperiorSkyblock2#1717] [Pending Feature Requests]
This command allows you to bypass the limit on island members, so I think it's worth giving it a separate permission to regulate this feature

**Comment by exzolink** [comment]
Wanna suggest same for /is balance nickname

**Comment by OmerBenGera** [comment]
How does it let you bypass members limit?

**Reply by exzolink** [reply]
> How does it let you bypass members limit?

They grant all the necessary perms to bypass teleport on closed island, building perm, interaction, etc.

**Reply by OmerBenGera** [reply]
> > How does it let you bypass members limit?
> 
> They grant all the necessary perms to bypass teleport on closed island, building perm, interaction, etc.

Island privileges does not let you bypass members limit. Members limit is enforced on all islands equally.

**Reply by exzolink** [reply]
> > > How does it let you bypass members limit?
> > 
> > 
> > They grant all the necessary perms to bypass teleport on closed island, building perm, interaction, etc.
> 
> Island privileges does not let you bypass members limit. Members limit is enforced on all islands equally.

They can act as island players (exclude bank, upgrades etc) and there is no need to add to coop even

**Reply by exzolink** [reply]
I do not mean bypassing the limit in the literal sense

**Comment by Lemvs** [comment]
@exzolink added as edit-player-permissions option in config.
@OmerBenGera you can close it.
